chapters/packaging.Rmd

```diff
@@ -648,63 +622,79 @@ and will vary from computer to computer.
 
 ## Distributing Packages {#packaging-distribute}
 
+> **Look But Don't Execute**
+>
+> In this section we upload the `pyzipf` package
+> to TestPyPI and PyPI:
+>
+> <https://test.pypi.org/project/pyzipf>
+>
+> <https://pypi.org/project/pyzipf/>
+>
+> You won't be able to execute the `twine upload`
+> commands exactly as shown
+> (because Amira has already uploaded the `pyzipf` package),
+> but the general sequence of commands in this section
+> is an excellent resource to refer back to when you are
+> uploading your own packages.
```

Comment:
```suggestion
> uploading your own packages.
> If you want to try uploading your own package via `twine`,
> you could suffix it with your name: `pyzipf-yourname`
```

Reply:
I wasn't sure whether to encourage people to clog up TestPyPI and PyPI with their own copy of the zipf package or not? I guess it doesn't really matter if they do. We could encourage them to delete their copy when they're done trying it out?

Reply:
I had the same thought, maybe recommend it for test-pypi specifically? I updated my suggestion.

Reply:
TestPyPI should be cleaned every now and then, so I don't think they need to delete anything https://packaging.python.org/guides/using-testpypi/#registering-your-account